packages/fast-data-utilities-react/src/__tests__/schemas/any-of.schema.json

```diff
@@ -0,0 +1,93 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Component with anyOf",
```

Comment:
i would describe what anyOf is in the title rather than speaking in code terminology.